	@Test(expected = ConversionException.class)
	public void invalidIntegerConversion3()
	{
		IntegerConverter converter = new IntegerConverter();
		converter.convertToObject("1.0", Locale.US);
		fail("Conversion should have thrown an exception");
	}

